### DISCUSSION TITLE
Redirection URL https://myonwnsite.mydom.com/ is safe ... but is a fail

### DISCUSSION THREAD

**Discussion by Nomtest** [authelia/authelia#4473] [Q&A] [answered]
I can't see why an allowed access come back immediately to the login screen again with anonymous, whitout opening the target site requested.

Note : the target site requested has already this user and the same password in its own authent so it's should match the authorisation.

I can't undersanb but experimented people may know this behavior...?

time="2022-12-04T19:29:34+01:00" level=debug msg="Mark 1FA authentication attempt made by user 'kate.familyname'" method=POST path=/api/firstfactor remote_ip=192.168.1.1
time="2022-12-04T19:29:35+01:00" level=debug msg="Successful 1FA authentication attempt made by user 'kate.familyname'" method=POST path=/api/firstfactor remote_ip=192.168.1.1
time="2022-12-04T19:29:35+01:00" level=debug msg="Check authorization of subject username=kate.familyname groups= ip=192.168.1.1 and object https://myonwnsite.mydom.com/ (method GET)."
time="2022-12-04T19:29:35+01:00" level=debug msg="Required level for the URL https://myonwnsite.mydom.com/ is 1" method=POST path=/api/firstfactor remote_ip=192.168.1.1
time="2022-12-04T19:29:35+01:00" level=debug msg="Redirection URL https://myonwnsite.mydom.com/ is safe" method=POST path=/api/firstfactor remote_ip=192.168.1.1
time="2022-12-04T19:29:35+01:00" level=debug msg="Check authorization of subject username= groups= ip=192.168.1.1 and object https://myonwnsite.mydom.com/ (method GET)."
time="2022-12-04T19:29:35+01:00" level=info msg="Access to https://myonwnsite.mydom.com/ (method GET) is not authorized to user <anonymous>, responding with status code 302 with location redirect to https://authelia-server/auth/?rd=https%3A%2F%2Fmyonwnsite.mydom.com%2F&rm=GET" method=GET path=/api/verify remote_ip=192.168.1.1

**Comment by james-d-elliott** [comment, marked as answer]
This is indicative of a bad proxy configuration. `https://authelia-server/auth/?rd=https%3A%2F%2Fmyonwnsite.mydom.com%2F&rm=GET` should be `https://auth.mydom.com/?rd=https%3A%2F%2Fmyonwnsite.mydom.com%2F&rm=GET` or similar. It must be the domain or a subdomain of the domain configured in:

```yaml
session:
  domain: mydom.com
```
